blaze-client/src/main/scala/org/http4s/client/blaze/PooledHttp1Client.scala

```diff
@@ -18,10 +18,12 @@ object PooledHttp1Client {
       maxWaitQueueLimit: Int = bits.DefaultMaxWaitQueueLimit,
       maxConnectionsPerRequestKey: RequestKey => Int = _ => bits.DefaultMaxTotalConnections,
       config: BlazeClientConfig = BlazeClientConfig.defaultConfig): Client[F] =
-    Http1Client.mkClient(
-      config.copy(
-        maxTotalConnections = maxTotalConnections,
-        maxWaitQueueLimit = maxWaitQueueLimit,
-        maxConnectionsPerRequestKey = maxConnectionsPerRequestKey
-      ))
+    Effect[F].toIO {
+      Http1Client.mkClient(
+        config.copy(
+          maxTotalConnections = maxTotalConnections,
+          maxWaitQueueLimit = maxWaitQueueLimit,
+          maxConnectionsPerRequestKey = maxConnectionsPerRequestKey
+        ))
+    }.unsafeRunSync()
```

Comment:
If we're going to force `toIO`, might as well make the whole call return `F[Client[F]]` instead. There's no need to maintain source compat on a deprecated method since M7 of 0.18.

Reply:
@jmcardon Is it okay to just remove the `PooledHttp1Client` then? I only did this so that we could keep it around, but figured I'd get questions about it.